packages/elements/src/components/ino-datepicker/type-specific-options.ts

```diff
@@ -0,0 +1,101 @@
+import monthSelectPlugin from 'flatpickr/dist/plugins/monthSelect';
+import { BaseOptions } from 'flatpickr/dist/types/options';
+
+export type PickerTypeOptions = 'date' | 'month' | 'time' | 'datetime';
```

Comment:
I had to use a workaround because you cannot extract keys from enums pre TS 4.1 (see [here](https://stackoverflow.com/questions/52393730/typescript-string-literal-union-type-from-enum)) but it's still an improvement